Dockerfile.tools

```diff
@@ -32,6 +32,10 @@ RUN pip2 install ansible pywinrm
 # Make Ansible happy with arbitrary UID/GID in OpenShift.
 RUN chmod g=u /etc/passwd /etc/group
 
+# Allow running WSU
```

Comment:
Please explain why this allows WSU to run. If it is because `go test` relies on the cache being set, and the user running the tests in our CI cluster doesn't have the permissions to create it at runtime, call it out explicitly.